mappings/net/minecraft/util/collection/FilterableCollection.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_5572 net/minecraft/util/collection/FilterableCollection
```

Comment:
Given this involves chunk statuses, it probably makes sense for this name to be a bit more specific. Don't fully understand the scope of this class so don't have any specific propositions- but maybe it can be specified to be to do with entities/chunks